docs/source/usage.rst

```diff
@@ -386,7 +385,7 @@ Interpolated nodes can be any node in the config, not just leaf nodes:
 Environment variable interpolation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Environment variable interpolation is also supported.
+Access to environment variables is supported using ``oc.env``:
```

Comment:
At this stage we might be able to document it like a regular function and copy the doc here as is. Example:
```python
def env(key: str, default: Optional[str] = _EMPTY_MARKER_) -> Optional[str]:
   """
   :param key: environment variable to return 
   :param default: Default value to use if environment variable is not present (use the literal 'null' in the config to indicate None. If a default is not provided and the environment variable is not defined an exception is raised.
   :return: The environment variable string or the default value
   """
```

This along with the example looks like a good structure to document this and other built-in resolvers (Especially once we start to annotate the types in the resolvers).

Reply:
Created new issue for this: #619